validation-test/stdlib/RuntimeSymbols.swift

```diff
@@ -1,6 +1,6 @@
 # Verify symbols from libswiftCore.dylib.
 
-UNSUPPORTED: OS=linux
+UNSUPPORTED: OS=linux-gnu
```

Comment:
The clean way is to add %target-nm-imports and %target-nm-exports to lit.cfg. I'll do that.